edx_django_utils/logging/internal/filters.py

```diff
@@ -0,0 +1,39 @@
+"""
+Django-based logging filters
+
+UserIdFilter: A logging.Filter that adds userid to the logging context
+
+RemoteIpFilter: A logging filter that adds the remote IP to the logging context
```

Comment:
I would remove this because I imagine it could easily get out of sync with the classes in the file.

Reply:
Removed in latest.